docs/contributing/development-policies.rst

```diff
@@ -94,18 +94,16 @@ Branches
 
 We maintain a number of branches on `our GitHub repository <https://github.com/divio/django-cms>`_.
 
-the latest (highest-numbered) ``release/x.y.z``
-    This is the branch that will become the next release on PyPI.
-
-    **Fixes and backwards-compatible improvements** (i.e. most pull requests) will be made against
-    this branch.
-
 ``develop``
-    This is the branch that will become the next release that increments the ``x`` or ``y`` of the latest
-    ``release/x.y.z``.
+    The default target branch for on-going development.
+
+the latest (highest-numbered) ``release/x.y.z``
+    This is the branch that will become the next release on PyPI. ``develop`` will be merged into ``release/x.y.z``
+    regularly.
 
-    This branch is for **new features and backwards-incompatible changes**. By their nature, these will require more
-    substantial team co-ordination.
+``feature`` branches
```

Comment:
we don't really use this.
most development happens on forks outside of core

Reply:
don't think we need feature branches really at the moment. if only to show development intent to whoever is watching but I kinda doubt anyone is doing that really